region/norfair/east/Frog Speedway.json

```diff
@@ -162,13 +172,215 @@
         "It is possible to do this while crossing the room with SpeedBooster by jumping while blue at the end of the speed blocks."
       ]
     },
+    {
+      "link": [1, 1],
+      "name": "Frozen Beetom Moondance",
+      "requires": [
+        "canMoondance",
+        "canTrickyUseFrozenEnemies",
+        {"enemyDamage": {
+          "enemy": "Beetom",
+          "type": "contact",
+          "hits": 2
+        }}
+      ],
+      "exitCondition": {
+        "leaveWithStoredFallSpeed": {
+          "fallSpeedInTiles": 1
+        }
+      },
+      "note": "Freeze a Beetom at head height where Samus can Spinjump into it and begin Moondancing."
+    },
+    {
+      "link": [1, 1],
+      "name": "Frozen Beetom Extended Moondance",
+      "requires": [
+        "canExtendedMoondance",
+        "canTrickyUseFrozenEnemies",
+        {"enemyDamage": {
+          "enemy": "Beetom",
+          "type": "contact",
+          "hits": 2
+        }},
+        "h_ExtendedMoondanceBeetomLeniency"
+      ],
+      "exitCondition": {
+        "leaveWithStoredFallSpeed": {
+          "fallSpeedInTiles": 2
+        }
+      },
+      "note": [
+        "Freeze a Beetom at head height where Samus can Spinjump into it and begin Moondancing.",
+        "After 195 moonfalls, reposition the Beetom to chest height, then continue dancing."
```

Comment:
```suggestion
        "After 175 moonfalls, reposition the Beetom to chest height, then continue dancing."
```
There are several strats that say 195 like this; I assume it may be a typo that was copy-pasted?